.github/workflows/php.yml

```diff
@@ -0,0 +1,33 @@
+name: PHP Composer
+on: [push, pull_request]
+jobs:
+  build:
+    name: Run tests on ${{ matrix.php }}
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [ '8.2', '8.3', '8.4' ]
```

Comment:
We should check down to the minimum version in composer.json here, and use PHPUnit 10 to support it.

Reply:
I rebased now and changed the first commit to require PHPUnit 10 and test on PHP 8.1+.